include/real/real_math.hpp

```diff
@@ -0,0 +1,298 @@
+#ifndef BOOST_REAL_MATH_HPP
+#define BOOST_REAL_MATH_HPP
+
+#include <tuple>
+#include "real/exact_number.hpp"

```

Comment:
can you reference all this includes using the <> notation?

Reply:
Done that. Will be merged in next commit.